features/enable_fips_cloud.feature

```diff
@@ -35,93 +33,29 @@ Feature: FIPS enablement in cloud based machines
            | fips          |
```

Comment:
Hasn't this changed, i.e. azure xenial has a fips kernel now?

Reply:
@CalvoM doesn't look like it. If we take a look at the fips repository here:
https://esm.ubuntu.com/fips/ubuntu/dists/xenial/main/binary-amd64/

We cannot see the `ubuntu-azure-fips` package. But let me know if I am missing something here

Reply:
Renan and Grant clarified about this during the backlog meeting. So it is okay.